lms/djangoapps/grades/rest_api/v1/gradebook_views.py

```diff
@@ -529,7 +529,11 @@ def get(self, request, course_key):
 
         if request.GET.get('username'):
             with self._get_user_or_raise(request, course_key) as grade_user:
-                course_grade = CourseGradeFactory().read(grade_user, course)
+                collected_block_structure = course_data.collected_structure
+                course_grade = CourseGradeFactory().read(
+                    grade_user,
+                    course,
+                    collected_block_structure)
```

Comment:
codestyle: paren should be on its own line

Reply:
Ugh, I always forget.